docker-compose.demo.uffizzi.yml

```diff
@@ -0,0 +1,41 @@
+# This compose file is for demonstration only, do not use in prod.
+version: "3.9"
+
+x-uffizzi:
+  ingress:
+    service: app
+    port: 3001
+  continuous_previews:
+    deploy_preview_when_pull_request_is_opened: true
+    delete_preview_when_pull_request_is_closed: true
```

Comment:
Do we need to re-deploy each time a PR is opened or closed? Seems unnecessary to me as we are using `prerelease` build for the live demo

Reply:
These are not used. Will remove.